dist/styles/table.scss

```diff
@@ -1,13 +1,17 @@
+.mdc-data-table {
```

Comment:
Lets remove this and go back to fixed as I'm not a fan of scrolling when someone who has long file names because of folder structure it is hard to see anything on 15" screens without scrolling.

Let's go back to fixed and adjust the width % that are used for column headers, I have tested and that resolves the issue.

Reply:
No prob as well.  We can agree to disagree about scrolling.  I see your argument.  I believe that you can't always know the user's browser environment so the flexibility afforded by letting the table do what it wants and we allow for scrolling, if necessary, is more user-friendly.  I also think some work should be done to make the top summary boxes more responsive.  There's text overlap at narrower widths there too at narrower widths.  I tried, but I don't know Material Design enough to do it correctly.  I'm not suggesting a mobile-friendly design because that's an unlikely use case.

All that said just for discussion among web developers. 😄 I will undo my change here.

Reply:
Regarding the sort function change… you're right.  I didn't consider that this method is not only used in the % coverage column.  I have accounted for that in my next update.  I've added a data attribute `data-sort-val` on each of the cells and that is the raw, unformatted value.  We sort on that value instead of the human-friendly `textContent` value.  This way we have a raw, unformatted, value that we also don't need to `trim()`.

For sorting the filename column the numeric sort option doesn't break the sort.  It sorts just like it would as a string comparison.

Reply:
See comment at bottom of conversation thread